### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please don't use git clone. Note also that corpus -> testsuite.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+image: ghcr.io/capuanob/str-fuzz:latest
```
Comment (xansec) [inline Mayhemfile:1]: Please omit image name.

File: fuzz.c [hunk 3]
```diff
@@ -0,0 +1,573 @@
+/*
```
Comment (xansec) [inline fuzz.c:1]: Can we rename this to indicate what's being fuzzed?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec I've made the requested changes!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**